test/functional/backends/test_netssh.rb

```diff
@@ -131,7 +131,7 @@ def test_execute_raises_on_non_zero_exit_status_and_captures_stdout_and_stderr
 
       def test_test_does_not_raise_on_non_zero_exit_status
         Netssh.new(a_host) do |_host|
-          test :false
+          test :false # rubocop:disable Lint/BooleanSymbol
```

Comment:
🗒️ RuboCop thinks this is a typo, and that we mean `false` and not `:false`. However in this case `:false` is the name of a command we are executing via SSH, not a boolean value.